content/20.pharmaceuticals.md

```diff
@@ -790,25 +790,45 @@ Dendritic cells are potent antigen-presenting immune cells that easily take up m
+On November 16, 2020, a report was released describing the initial results from Phase III testing, corresponding to the first 95 cases of COVID-19 in the 30,000 enrolled participants [@url:https://investors.modernatx.com/news-releases/news-release-details/modernas-covid-19-vaccine-candidate-meets-its-primary-efficacy], with additional data released to the FDA on December 17, 2020 [@url:https://www.fda.gov/advisory-committees/advisory-committee-calendar/vaccines-and-related-biological-products-advisory-committee-december-17-2020-meeting-announcement#event-materials].
+The first group of 30,351 study participants were administered the vaccine or a placebo at a ratio of 1:1 at 99 sites within the United States in two sessions, spaced 28 days apart [@url:https://www.fda.gov/media/144434/download].
+Patients reporting COVID-19 symptoms upon follow-up were tested for SARS-CoV-2 using a nasopharyngeal swab that was evaluated with RT-PCR [@url:https://www.fda.gov/media/144434/download].
+Of the first 95 cases reported, 90 occurred in participants receiving the placebo, compared to 5 cases in the group receiving the vaccine [@url:https://investors.modernatx.com/news-releases/news-release-details/modernas-covid-19-vaccine-candidate-meets-its-primary-efficacy].
+These results suggest the vaccine is 94.5% effective in preventing COVID-19.
+Additionally, eleven severe cases of COVID-19 were observed, and all eleven occurred in participants receiving the placebo.
+These findings suggest the possibility that the vaccine might bolster immune defenses even for subjects who do still develop a SARS-CoV-2 infection.
+The study was designed with an explicit goal of including individuals at high risk for COVID-19, including older adults, people with underlying health conditions, and people of color [@url:https://investors.modernatx.com/news-releases/news-release-details/moderna-has-completed-case-accrual-first-planned-interim].
+The Phase III trial population was comprised by approximately 25.3% adults over age 65 and 22.3% adults with one or more underlying condition [@url:https://www.fda.gov/media/144434/download]. 
+Among the 95 cases reported in the first interim analysis, 16% occurred in older adults [@url:https://investors.modernatx.com/news-releases/news-release-details/modernas-covid-19-vaccine-candidate-meets-its-primary-efficacy].
+Additionally, 9.7% of participants identified a Black or African-American background as well as 20% who identified Hispanic or Latino ethnicity [@url:https://www.fda.gov/media/144434/download].
+Among the 95 cases, 12.6% occurred in participants identifying a Hispanic or Latino background and 4% in participants reporting a Black or African-American background [@url:https://investors.modernatx.com/news-releases/news-release-details/modernas-covid-19-vaccine-candidate-meets-its-primary-efficacy].
```

Comment:
Should this be past-tense? maybe not just checking

Reply:
I'm not quite sure that sentence had any tense 😆 Should be fixed now to be at least vaguely grammatical!

Reply:
That happens haha